test/test_execjs.rb

```diff
@@ -376,4 +376,30 @@ def test_uglify
     assert_equal "function foo(bar){return bar}",
       context.call("uglify", "function foo(bar) {\n  return bar;\n}")
   end
+
+  def test_compiler_writing_to_stdout
```

Comment:
Maybe `stdio` instead of `stdout` since this is testing both `stdout` and `stderr`?